src/components/calcite-block-section/calcite-block-section.tsx

```diff
@@ -92,6 +98,19 @@ export class CalciteBlockSection {
   //  Render Methods
   //
   // --------------------------------------------------------------------------
+  renderStatusIcon(): VNode[] {
+    const { status } = this;
+    const statusIcon = ICONS[status] ?? false;
+    const statusIconClasses = {
+      [CSS.statusIcon]: true,
+      [CSS.valid]: status == "valid",
+      [CSS.invalid]: status == "invalid"
+    };
+
+    return !!statusIcon ? (
+      <calcite-icon class={statusIconClasses} icon={statusIcon} scale="s"></calcite-icon>
```

Comment:
nitpick. self close the `calcite-icon`. `<calcite-icon class={statusIconClasses} icon={statusIcon} scale="s" />`

Reply:
fixed.